### ISSUE TITLE
如何面对学术进化

### ISSUE THREAD

**Issue by wangaisheng** [steam-maker/steam-maker-forum#8]
我把在一个学术圈的发言总结在这里，与大家讨论如何面对学术进化。
1.项目学习
        关于项目学习，对学生来说，要放小一点，不要追求太高大上。从实验来看，做的效果不错，我也做过很多次，有案例。 项目学习与STEM不是一个概念。  还是那句话，要指望一个东东抱打天下，彻底革命是不可能的。  研究性学习也有相通之处，但是也要真研究。

2.个人进步
       我在十几年前写过一个文章《给计算机教师支什么招》说的是老师自己的定位问题，要拯救课程首先要发展教师，不论你做什么，要做出一些成绩来，并且尽量能够达到一定的水准。 比如说，写教材吧，你可以反对，但你写出自己的思路来啊，我就是这样做，所以从只写一个二进制做了到主编。 要做！一定要做！不做，永远是在踏步。
       我已经用自己的时光证明了：做课件也好，做课题也好，写论文也好，做教材也吧，上课也好，要去做，做出自己的特色。尽量不要跟风，至少在跟风中有自我。多少年过去了，如果只是在怀疑，那也没特别大的用途，怀疑是为了建设。我的学生说不想做ICT了，想教物理，我说好，做到极致，物理现在是英才班的班主任，在应试教育上得过很多政府奖。我另一个小同事，人老实80后，做事刻苦，现在也是电教中心主任了，他说起来就是认为我当年逼迫他做工作受益大。

3.多元化的时代性
     面对新课程，我们要去做，除非不想在这个方面混，有其他的爱好方向也非常好。但是，生命的状态在于运动。在目前的民主生态下，我们要促进民众的智慧，更有自我。简单地“暴动”或“霸权”，现在已经没有出入了。就像当年日本，它没有认识到已经进入后殖民时代了，盲目跟随了法西斯，只是促进了中国的觉醒，又是“客观上”引起了中国的进步与发展。 日本的理论界想起来很后悔，如果更多精力去做经济建设的话，走另外一条路可能日本已经是巨龙了。但是，历史没有如果，再想什么也已经是晚了。
      同样，想创客教育、STEM、项目学习、翻转课堂，像以前的目标学习、建构主义那样再有辉煌，对不起，已经晚了，因为在后课标时代，已经没有人再会以课标独尊，多元化、个性化的局面已经打开，谁也别想独霸教育了。  这就是翻转课堂等一切新名词的宿命论。 

4.教育职业化
         就像，再想回到“尊师重教”的单纯时代已经不可能了，希望老师自己安好，因为只是职业而已。那些打老师之类的事件，教师到了最危险的时候，也只是最后的哀鸣。教师就是一种职业，有职业风险，也有职业优势，认清自己，理性投资，哈~

以 上都是个人观点，供讨论。

**Comment by ghost** [comment]
很有見解。

教書育人和愚人一字之差。無論是什麼樣的環境和工具，人都是核心。

### 在信息大爆炸，什麼東西都可以自學的時代，還要教師做什麼？

![image](https://cloud.githubusercontent.com/assets/951921/14949232/33381d00-1016-11e6-81be-8ce20db2655c.png)

**Comment by xdq73** [comment]
【同样，想创客教育、STEM、项目学习、翻转课堂，像以前的目标学习、建构主义那样再有辉煌，对不起，已经晚了。】真正的学术进化不是这样后浪拍前浪的，而是继承与发展，继承占主导，发展往往都是某个点的小突破。这种你方唱罢我登场的形式，哪里是学术研究的套路，整个唱大戏凑热闹而已。

不认可教育教学领域上也存在多元化。如果教育教学是一种规律，是一种科学，不可能多元化并存的，必定存在最致命的一种武器或者武功。

多元化的世界，是价值观的多元化，如果把学术问题、科学问题也多元化了，那就有问题了。如果治病也多元化，想想挺可怕。一种病，必定存在一种最佳疗法，找不到另说，但如果说，可以多元化并存，不能理解。

教无定法是没有科学教育观念下的陈旧看法，科学教学论恰恰要否定的就是教无定法。人的认知是客观规律的话，教就是要适合认知规律，怎么可能无定法？无定法，恰恰说明不懂认知规律。要证明教无定法成立，就要推翻认知规律的客观性，证明每个人都有自己独特的一套认知规律，不用每个人，或者每一类人都有一类独特的认知规律。不幸的是，目前的认知研究、脑成像研究，都认为认知的规律是客观一致的。连大脑功能的部位都是一样的。建立在这样的认知基础上的教育，却要搞教无定法，很分裂。

**Comment by ghost** [comment]
@xdq73 

同意。站在巨人的肩膀上是科學發展的核心。另外也要把學派和真正意義上的問題分開。由於其主觀性，人文科學比自然科學更容易受到潮流的影響。

和其它進化論一樣，學術進化也是一個適者生存的原則。但是區別在於在這樣的虛擬世界，人對環境的影響也很大。所以不求甚解，不看具體實際情況，人云亦云，跟風現象很容易流行。直到真正的大環境改變到不能適應之後，下一輩創新者就會自然把接力棒奪走，超過去。

**Comment by ghost** [comment]
在我的[關於STEM的理解](https://www.digital-message.com/learn/STEM/)中，提到了Alan Kay的視角和智商的關係，還有一些歷史文獻，建議大家看看。拋開熱點名詞，仔細根據自己的實際經驗思考一下理論和學派的關係。自稱學派比看懂文獻要容易多了。

**Comment by wangaisheng** [comment]
做！实践，这是我个人提倡的。尤其是一线，只是理论推理，十年，二十年，时光荏苒没有实践，所谓专家可以这样，我们不行，都不做事都在谈话中，生产力哪里来？
